docs/team/jaivigneshvenugopal.adoc

```diff
@@ -0,0 +1,101 @@
+
+Thus, it is a requirement to have access to a list of blacklisted people. That is essentially what the `blacklist` command does. Moreover, working in the blacklist gives exclusive and focused control over the debtors in this list alone. Meaning, when working in the blacklist, users will not be able to access non-blacklisted persons. This is because, the commands work exclusively on blacklisted debtors alone.
+
+To access other lists, specific commands have to be used to toggle.
+
+==== Justification (Whitelist)
+
+As a debt collector, it is also important to identify and categorize debtors who have repaid all their debts. Ultimately, that is the purpose of the `whitelist` command. Debt collection agencies regularly need to liase with client banks to submit reports on debtors' activities and repayment amounts. Thus, having a list of debtors who have repaid their debts, along with the date that they repaid the full amount, essentially helps to track debtors' activities.
+
+If the debt collection agency happen to be a subsidiary institution to a bank, this whitelist feature will also aid in allowing the voice-call sectors to contact people for additional loan-signups.
+
+As aforementioned, to access other lists, specific commands have to be used to toggle.
+
+==== Implementation
+
+---
+#Start of Extract [from: Developer Guide]#
+
+include::../DeveloperGuide.adoc[tag=list]
+
+#End of Extract#
+
+---
+
+=== Enhancement Added: Repaid command
+
+==== External behavior
+
+---
+#Start of Extract [from: User Guide]#
+
+include::../UserGuide.adoc[tag=repaid]
+
+#End of Extract#
+
+---
+
+==== Justification
+
+A debt collector will have the job to record down the collections for the day from each debtor. He has to do so for each debtor. If a debtor has paid off his/her total debt, there should be an easy way for the debt collector to update this amount, date repaid and the debt flag of the debtor easily. That is the purpose of the Repaid Command.
+
+This is very similar to the Payback command, however, Repaid command excels in automating three different tasks so that life is indeed simpler for debt collectors to key in information in the system.
+
+=== Enhancement Added: Add/Remove Profile Picture
+
+==== External behavior
+
+---
+#Start of Extract [from: User Guide]#
+
+include::../UserGuide.adoc[tag=profilepic(soon to appear)]
+
+#End of Extract#
+
+---
+
+==== Justification
+
+A debt collector might need to conduct house visits to collect debts from people. Thus, having commands that allow the user to add/remove pictures for each individual debtors helps him/her identify and mentally register a specific debtor.
```

Comment:
i don't think "mentally register" is the right phrase to use here, maybe you can get rid of it or replace it with something else

Reply:
Noted